crates/unleash-edge-cli/src/lib.rs

```diff
@@ -533,53 +533,55 @@ impl CorsOptions {
     pub fn middleware(&self) -> CorsLayer {
         CorsLayer::new()
             .max_age(MaxAge::exact(Duration::from_secs(self.cors_max_age)))
-            .allow_headers(Any)
```

Comment:
I think this is redundant, since we call `.allow_headers(self.allowed_headers())` a few lines down.